Automattic/wp-calypso
# Stats: Rename PromoCard component

#### Proposed Changes

Renames the PromoCard component to JetpackMobilePromoCard. The repository already has two other more generic components named PromoCard.

#### Testing Instructions



* Check out this branch locally.
* Spin up Storybook via `yarn workspace @automattic/components run storybook`.
* Check the component shows up as Jetpack Mobile Promo Card now.
* Confirm it behaves as expected.

#### Pre-merge Checklist





Related to #69230
